include/deal.II/base/vectorization.h

```diff
@@ -2707,6 +2707,25 @@ vectorized_transpose_and_store(const bool                    add_into,
 
 
 /**
+ * Relational operator == for VectorizedArray
+ *
+ * @relates VectorizedArray
+ */
+template <typename Number>
+inline DEAL_II_ALWAYS_INLINE
+bool
+operator == (const VectorizedArray<Number> &lhs,
```

Comment:
I guess we never tested `operator=(Number)` before. I guess you stumbled over a test that exposed this? In any case, it makes definitely sense to add this operator overload. :+1: